### ISSUE TITLE
Contribution: Xcode template support

### ISSUE THREAD

**Issue by jcampbell05** [rambler-digital-solutions/Generamba#152]
So I wanted to make something similar to this at 
https://github.com/jcampbell05/cook

But I was wondering whats the best way to contribute? I want to implement something that can use existing Xcode templates as there is already an ecosystem.

Would you be open to me adding support for that ? Using this in combination with Xcake will be amazing!

**Comment by etolstoy** [comment]
Hi! @jcampbell05! Can you explain what exactly you want to achieve?

**Comment by jcampbell05** [comment]
@etolstoy There is a community of good Xcode Templates already out there and rather than build yet another generator I just thought it owuld be great if Generamba allowed you to scaffold a new template using them.

**Comment by etolstoy** [comment]
@jcampbell05 One of the reasons why we decided to create Generamba was the ugly and inconvenient syntax of Xcode templates. I covered it briefly in [this article](http://etolstoy.com/2016/02/10/generamba/). 

Besides it, using liquid, it's possible to do really great things in templates - e.g. using conditional statements, loops, mixins and so on. None of these features are available with Xcode templates.
